exporter/datadogexporter/examples/sdk-go/go.mod

```diff
@@ -0,0 +1,31 @@
+module sdk-example
```

Comment:
Anything other than Collector YAML configuration samples should not be in this repository (see #7155 and #11074 for precedent)